x-pack/test/accessibility/apps/remote_clusters.ts

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+import { ClusterPayloadEs } from '../../../plugins/remote_clusters/common/lib';
+
+const emptyPrompt = 'remoteClusterListEmptyPrompt';
+const createButton = 'remoteClusterEmptyPromptCreateButton';
+const pageTitle = 'remoteClusterPageTitle';
+const nameLink = 'remoteClustersTableListClusterLink';
+const editButton = 'remoteClusterTableRowEditButton';
+const deleteButton = 'remoteClusterTableRowRemoveButton';
+const deleteModalTitle = 'confirmModalTitleText';
+const detailsTitle = 'remoteClusterDetailsFlyoutTitle';
+const requestButton = 'remoteClustersRequestButton';
+const requestTitle = 'remoteClusterRequestFlyoutTitle';
+
+interface Payload {
+  persistent: {
+    cluster: {
+      remote: {
+        [k: string]: ClusterPayloadEs;
+      };
+    };
+  };
+}
+
+const getEmptyPayload = () => ({
+  persistent: {
+    cluster: {
+      remote: {},
+    },
+  },
+});
+
+const getPayloadClusterProxyMode = (name: string): Payload => {
+  const payload = getEmptyPayload() as Payload;
```

Comment:
nit: you could cast it only once  on L38 so you don't have to cast it each time you consume it